Header header
int32 index
bool engage
float32 velocity
float32 acceleration
int64 position
bool reset_position
